L1Trigger/Phase2L1ParticleFlow/python/l1tSeedConePFJetProducer_cfi.py

```diff
@@ -1,12 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-l1tSeedConePFJetProducer = cms.EDProducer("L1SeedConePFJetProducer",
-                           L1PFObjects = cms.InputTag("l1tLayer1","Puppi"),
-                           nJets       = cms.uint32(10),
-                           coneSize    = cms.double(0.4),
-                           HW          = cms.bool(False),
-                           debug       = cms.bool(False)
-                         )
+l1tSeedConePFJetProducer = cms.EDProducer("L1SeedConePFJetProducer")
```

Comment:
```suggestion
from L1Trigger.Phase2L1ParticleFlow.l1tSeedConePFJetProducer_cfi import l1tSeedConePFJetProducer
```
if in https://github.com/cms-sw/cmssw/pull/41279/files#diff-fbe4efdfa3a3a6b083dd6fa21bfe8aa555e61bfa1863c36ccdbb0000f6d4fbd8R218 you have named the cfi created by the `fillDescriptions` method `1tSeedConePFJetProducer`

Otherwise, import from it and clone `1tSeedConePFJetProducer` from that config